assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -258,7 +262,8 @@ const MediaElement = ({
     width,
     height,
     onClick,
-    showVideoDetail,
+    showVideoDetail: showVideoDetail && !showAttribution,
```

Comment:
maybe you can roll up the extra logic into the useEffect where setShowVideoDetail is called? That way showVideoDetail truly means what its name implies.

Reply:
Done. But I created a separate setShowVideoDuration and a separate effect.